model_evaluation/rfc.md

```diff
@@ -0,0 +1,91 @@
+### What are the assets being proposed?
+**Component**
+  Model Evaluation component
+
+**Environment**
+  Custom Environment for Model Evaluation
+  Command component specific environment will be shared, which will contain following assets: 
+
+  1. [Dockerfile](https://microsoft.sharepoint.com/:u:/t/SDAutoML/EbYHBgGmkmNNjUIvWEyci3gBT-Uqu73nwf5UPMj02BP0ow?e=G5wxcE)
+  2. env.yaml
+
+  
+### Why should this asset be built-in?
+
+Model Evaluation would be a part of DPV2 which allows user to run Model Evaluation for Any Machine Learning model on Azure ML.  In order for us to make the functionality of model evaluation available as generic functionality at Azure Machine Learning level, this component should be available as built-in component which users should be able to use it as drag and drop in designer.
+
+Model Evaluation will be a standalone component available to user in all 3 user-experience. Currently, we have implemented the model evaluation feature as Command Component. SDK and CLI internally create a pipeline job which consumes model evaluation component. 3 user experiences as below: 
```

Comment:
Will the evaluation component support distributed scoring? It would be a requirement for scalable model evaluation.

Reply:
The distributed scoring and batch inferencing functionality is planned for Post PrP.